dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -330,6 +333,53 @@ def kurtosis(self):
                     self.match_count,
                     self._biased_kurtosis)
 
+    @staticmethod
+    def _perform_t_test(mean1, var1, n1,
+                        mean2, var2, n2):
+        results = {}
+        s_delta = var1/n1 + var2/n2
+        t = (mean1 - mean2) / np.sqrt(s_delta)
+        conservative_df = min(n1, n2) - 1
+        welch_df = s_delta ** 2 / ((var1 / n1) ** 2 / (n1 - 1) + (var2 / n2) ** 2 / (n2 - 1))
+        results.update({
+            "t-statistic": t,
+            "conservative_df": conservative_df,
+            "welch_df": welch_df
+        })
+        try:
+            import scipy.stats
+        except ImportError:
+            # Failed, so we return the stats but don't perform the test
+            warnings.warn("Could not import necessary statistical packages. "
+                          "T-test will be incomplete.", RuntimeWarning)
```

Comment:
at the very least should say `pip install scipy` best case scenario we abstract `warn_on_profile` to be utilized for both profiling or any case where the user needs to install a set of requirements like `dataprofiler[ml]` or if we do `dataprofiler[stats]`